lib/utils.js

```diff
@@ -24,12 +24,38 @@ function sendSearchResponse(movies, resp) {
 
   resp
     .say(['Add', movies[0].original_title, '(' + movies[0].year + ')', 'to your list?'].join(' '))
-    .session('promptData', buildPrompt(movies.slice(0, 5)))
+    .session('promptData', buildPrompt(movies))
     .shouldEndSession(false)
     .send();
 }
 
+function formatSearchResults(movies) {
+  
+    if (movies != undefined) {
+
+      var newMovies = [];
+
+      for (var i = 0; i < movies.length; i++) {
+
+        newMovies.push({
+          original_title: movies[i].original_title,
+          inLibrary: isInLibrary(movies[i].in_library),
```

Comment:
I added a new function to check if the movie is already in your library (different json is defined) but I didn't commit that function - removed the reference to it.